### PR TITLE
implemented PEP-8 standards

### PR DESCRIPTION
**Pull request by newbazz**

### CODE REVIEW

**Comment by vaibhavsingh97** [comment]
@newbazz too many commits. Please squash into one 😄

**Comment by vaibhavsingh97** [comment]
@newbazz Thanks for your effort to fix issues. PLease keep in mind that before working in any project please fetch latest codebase then start working on project. Also, it's a good practice if you make new branch for every issue you are fixing.

**Comment by Pk13055** [comment]
Please refer to #151. And next time onwards, do take a look at the existing pulls before creating a new one!

**Comment by vaibhavsingh97** [comment]
@newbazz Please refer the changes suggested by me in #152.

File: .bowerrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "directory" : "app/static/bower_components"
+  "directory" : "app/static/bower_components", 
+  "proxy": "http://proxy.iiit.ac.in:8080",
+  "https-proxy":"http://proxy.iiit.ac.in:8080",
+  "no-proxy":"myserver.mydomain.com"
```
Comment (vaibhavsingh97) [inline .bowerrc]: extraneous change. Please add changes relevant to PR

**Comment by AnshulMalik** [comment]
Redundant with #211, should be closed

**Comment by vaibhavsingh97** [comment]
@mariobehling PLease close this PR as it's already been fixed in the recent merge

**Comment by cclauss** [comment]
@newbazz Can you please close this PR.  The codebase is now PEP8 compliant and we now automatically fail all PRs that are not PEP8 compliant.  See https://github.com/fossasia/query-server/blob/master/.travis.yml#L9

**Comment by cclauss** [comment]
PLEASE CLOSE!

**Pull request closed without merging**